e2e/test/scenarios/admin/performance/performance.cy.spec.ts

```diff
@@ -1,46 +1,17 @@
 import { describeEE, restore, setTokenFeatures } from "e2e/support/helpers";
 
```

Comment:
I've moved these helper functions to a separate file and changed them a bit, so these tests need to change a bit